.ci/azure/linux.yml

```diff
@@ -167,24 +168,44 @@ jobs:
   - script: |
       python3 -m pip install --user virtualenv
       python3 -m virtualenv -p /usr/bin/python3.8 .env3
-      source .env3/bin/activate
+      source .env3/bin/activate && source $(SETUPVARS)
       python -m pip install --upgrade pip
       python -m pip install -r $(REPO_DIR)/modules/custom_operations/tests/requirements.txt
       python -m pip install $(INSTALL_DIR)/tools/openvino-*.whl
       python -m pip install $(OPENVINO_REPO_DIR)/tools/mo/
-      python -m pip install $(REPO_DIR)/modules/custom_operations/user_ie_extensions/tokenizer/python/.[all]
     workingDirectory: $(WORK_DIR)
     displayName: 'Create virtual env'
 
   - script: |
-      source  $(WORK_DIR)/.env3/bin/activate
+      source $(WORK_DIR)/.env3/bin/activate && source $(SETUPVARS)
+      python -m pip install build
+      python -m build --wheel --outdir $(BUILD_WHEEL_DIR) $(REPO_DIR)/modules/custom_operations
```

Comment:
Should I now `pip install modules/custom_operations` to get tokenizers?

Please, update https://github.com/openvinotoolkit/openvino_contrib/blob/master/modules/custom_operations/user_ie_extensions/tokenizer/python/README.md

Reply:
CC @mryzhov to update docs

> Should I now pip install modules/custom_operations to get tokenizers?

Yes for Python API (conversion capabilities)
For C++ you either need to point out to some place inside site-packages or build extension using native cmake commands.